### ISSUE TITLE
Our CSS needs attention.

### ISSUE THREAD

**Issue by t4k** [Coral-erm/Coral#242]
Pulled out of the conversation in #77 so this can be worked on bit by bit with an issue to point pull requests at.

> It’s ad hoc, and not easily reusable or extensible. Part of this is due to the over-reliance on IDs serving as jquery hooks (which also tightly couples our HTML to our javascript to our CSS, making it difficult to improve any of the three). Another part is the abuse of the !important declaration, which breaks the C in CSS and is difficult to identify and fix.

**Comment by jcuenod** [comment]
Are you suggesting rewriting the frontend? This seems like an issue that can't be solved without a pretty significant overhaul...

**Comment by jsavell** [comment]
It could be addressed piece by piece in scoped PRs.

Here's a refactor of the Queue functionality I did awhile back:

https://github.com/Coral-erm/Coral/pull/143/files

Looking at the changes to style.css, it's just an incremental improvement on a small section of the code, but the css file shrank by _45 lines_.

Each incremental improvement will make the code easier to maintain and simplify any potential future front end rewrites.

**Comment by t4k** [comment]
Yes, my plan was to do some work similar to that mentioned in the comment above: small chunks of changes that will improve things over time.

Having this issue active will let us make PRs associated with it so we can group any CSS changes together.
